{
 "bad":{
  "exception":"<class 'requests.exceptions.ReadTimeout'>: A connection-level exception occurred: HTTPSConnectionPool(host='api.github.com', port=443): Read timed out. (read timeout=10)",
  "traceback":[
   "Traceback (most recent call last):",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/urllib3/connectionpool.py\", line 421, in _make_request",
   "    six.raise_from(e, None)",
   "  File \"<string>\", line 3, in raise_from",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/urllib3/connectionpool.py\", line 416, in _make_request",
   "    httplib_response = conn.getresponse()",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/http/client.py\", line 1321, in getresponse",
   "    response.begin()",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/http/client.py\", line 296, in begin",
   "    version, status, reason = self._read_status()",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/http/client.py\", line 257, in _read_status",
   "    line = str(self.fp.readline(_MAXLINE + 1), \"iso-8859-1\")",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/socket.py\", line 589, in readinto",
   "    return self._sock.recv_into(b)",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/urllib3/contrib/pyopenssl.py\", line 326, in recv_into",
   "    raise timeout(\"The read operation timed out\")",
   "socket.timeout: The read operation timed out",
   "",
   "During handling of the above exception, another exception occurred:",
   "",
   "Traceback (most recent call last):",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/requests/adapters.py\", line 449, in send",
   "    timeout=timeout",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/urllib3/connectionpool.py\", line 720, in urlopen",
   "    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/urllib3/util/retry.py\", line 400, in increment",
   "    raise six.reraise(type(error), error, _stacktrace)",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/urllib3/packages/six.py\", line 735, in reraise",
   "    raise value",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/urllib3/connectionpool.py\", line 672, in urlopen",
   "    chunked=chunked,",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/urllib3/connectionpool.py\", line 423, in _make_request",
   "    self._raise_timeout(err=e, url=url, timeout_value=read_timeout)",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/urllib3/connectionpool.py\", line 331, in _raise_timeout",
   "    self, url, \"Read timed out. (read timeout=%s)\" % timeout_value",
   "urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host='api.github.com', port=443): Read timed out. (read timeout=10)",
   "",
   "During handling of the above exception, another exception occurred:",
   "",
   "Traceback (most recent call last):",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/github3/models.py\", line 196, in _request",
   "    return request_method(*args, **kwargs)",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/requests/sessions.py\", line 581, in post",
   "    return self.request('POST', url, data=data, json=json, **kwargs)",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/github3/session.py\", line 169, in request",
   "    response = super(GitHubSession, self).request(*args, **kwargs)",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/requests/sessions.py\", line 533, in request",
   "    resp = self.send(prep, **send_kwargs)",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/requests/sessions.py\", line 646, in send",
   "    r = adapter.send(request, **kwargs)",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/requests/adapters.py\", line 529, in send",
   "    raise ReadTimeout(e, request=request)",
   "requests.exceptions.ReadTimeout: HTTPSConnectionPool(host='api.github.com', port=443): Read timed out. (read timeout=10)",
   "",
   "During handling of the above exception, another exception occurred:",
   "",
   "Traceback (most recent call last):",
   "  File \"/root/repo/cf-scripts/conda_forge_tick/auto_tick.xsh\", line 682, in main",
   "    hash_type=attrs.get('hash_type', 'sha256'))",
   "  File \"/root/repo/cf-scripts/conda_forge_tick/auto_tick.xsh\", line 130, in run",
   "    migrator.remote_branch())",
   "  File \"/root/repo/cf-scripts/conda_forge_tick/git_utils.xsh\", line 230, in push_repo",
   "    pr = repo.create_pull(title, 'master', head, body=body)",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/github3/decorators.py\", line 31, in auth_wrapper",
   "    return func(self, *args, **kwargs)",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/github3/repos/repo.py\", line 1117, in create_pull",
   "    return self._create_pull(data)",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/github3/repos/repo.py\", line 120, in _create_pull",
   "    json = self._json(self._post(url, data=data), 201)",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/github3/models.py\", line 221, in _post",
   "    return self._request(\"post\", url, data, **kwargs)",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/github3/models.py\", line 201, in _request",
   "    raise exceptions.ConnectionError(exc)",
   "github3.exceptions.ConnectionError: <class 'requests.exceptions.ReadTimeout'>: A connection-level exception occurred: HTTPSConnectionPool(host='api.github.com', port=443): Read timed out. (read timeout=10)",
   ""
  ]
 },
 "conda-forge.yml":{},
 "feedstock_name":"maestral",
 "hash_type":"sha256",
 "meta_yaml":{
  "about":{
   "dev_url":"https://github.com/SamSchott/maestral",
   "home":"https://github.com/SamSchott/maestral",
   "license":"MIT",
   "license_family":"MIT",
   "license_file":"LICENSE.txt",
   "summary":"Open-source Dropbox client for macOS and Linux."
  },
  "build":{
   "entry_points":[
    "maestral=maestral.cli:main",
    "maestral=maestral.cli:main",
    "maestral=maestral.cli:main"
   ],
   "noarch":"python",
   "number":"0",
   "script":" -m pip install . --no-deps --ignore-installed -vv "
  },
  "extra":{
   "recipe-maintainers":[
    "tschoonj",
    "tschoonj",
    "tschoonj"
   ]
  },
  "package":{
   "name":"maestral",
   "version":"0.4.1"
  },
  "requirements":{
   "host":[
    "pip",
    "python >=3.6",
    "setuptools",
    "pip",
    "python >=3.6",
    "setuptools",
    "pip",
    "python >=3.6",
    "setuptools"
   ],
   "run":[
    "blinker",
    "click >=7.0",
    "dropbox >=9.4.0",
    "keyring >=19.0.0",
    "keyrings.alt >=3.0.0",
    "pyqt",
    "pyro4",
    "python >=3.6",
    "requests",
    "setuptools",
    "u-msgpack-python",
    "watchdog",
    "blinker",
    "click >=7.0",
    "dropbox >=9.4.0",
    "keyring >=19.0.0",
    "keyrings.alt >=3.0.0",
    "pyqt",
    "pyro4",
    "python >=3.6",
    "requests",
    "setuptools",
    "u-msgpack-python",
    "watchdog",
    "blinker",
    "click >=7.0",
    "dropbox >=9.4.0",
    "keyring >=19.0.0",
    "keyrings.alt >=3.0.0",
    "pyqt",
    "pyro4",
    "python >=3.6",
    "requests",
    "setuptools",
    "u-msgpack-python",
    "watchdog"
   ]
  },
  "source":{
   "sha256":"eed1ad568ccd5cb16b42043537e97bc0bdedf122745279bd06902dc61e053425",
   "url":"https://pypi.io/packages/source/m/maestral/maestral-0.4.1.tar.gz"
  },
  "test":{
   "commands":[
    "maestral --help",
    "maestral --help",
    "maestral --help"
   ],
   "imports":[
    "maestral",
    "maestral",
    "maestral"
   ]
  }
 },
 "name":"maestral",
 "new_version":"0.4.2",
 "raw_meta_yaml":"{% set name = \"maestral\" %}\n{% set version = \"0.4.1\" %}\n\npackage:\n  name: \"{{ name|lower }}\"\n  version: \"{{ version }}\"\n\nsource:\n  url: \"https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz\"\n  sha256: eed1ad568ccd5cb16b42043537e97bc0bdedf122745279bd06902dc61e053425\n\nbuild:\n  number: 0\n  entry_points:\n    - maestral=maestral.cli:main\n  noarch: \"python\"\n  script: \"{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv \"\n\nrequirements:\n  host:\n    - pip\n    - python >=3.6\n    - setuptools\n  run:\n    - blinker\n    - click >=7.0\n    - dropbox >=9.4.0\n    - keyring >=19.0.0\n    - keyrings.alt >=3.0.0\n    - pyqt\n    - pyro4\n    - python >=3.6\n    - requests\n    - setuptools\n    - u-msgpack-python\n    - watchdog\n\ntest:\n  imports:\n    - maestral\n  commands:\n    - maestral --help\n\nabout:\n  home: \"https://github.com/SamSchott/maestral\"\n  license: \"MIT\"\n  license_family: \"MIT\"\n  license_file: \"LICENSE.txt\"\n  summary: \"Open-source Dropbox client for macOS and Linux.\"\n  dev_url: \"https://github.com/SamSchott/maestral\"\n\nextra:\n  recipe-maintainers:\n    - tschoonj\n",
 "req":{
  "__set__":true,
  "elements":[
   "blinker",
   "click",
   "dropbox",
   "keyring",
   "keyrings.alt",
   "pip",
   "pyqt",
   "pyro4",
   "python",
   "requests",
   "setuptools",
   "u-msgpack-python",
   "watchdog"
  ]
 },
 "url":"https://pypi.io/packages/source/m/maestral/maestral-0.4.1.tar.gz",
 "version":"0.4.1"
}